src/test/unit/common/services/cliConfigService.test.ts

```diff
@@ -19,14 +27,17 @@ suite('CLI Config Service', () => {
     };
 
     // act
-    await getSastSettings(apiClient);
+    await getSastSettings(apiClient, config);
 
     // assert
     strictEqual(
       getFake.calledWith(sinon.match.any, {
         headers: {
           'x-snyk-ide': `${Configuration.source}-${await Configuration.getVersion()}`,
         },
+        params: {
+          org: 'my-super-org',
```

Comment:
minor: you can refer to `config.organization` string here, instead of typing it twice

Reply:
makes sense, done with 1264a7e